### PR TITLE
DBZ-6192 - Add Database role support for Spanner Connector config.

### PR DESCRIPTION
**Pull request by imtj1**
Add Database role support for Spanner Connector config.

### CODE REVIEW

**Review by jpechane — APPROVED** [review]
@imtj1 LGTM, thanks! Waitiing for @nancyxu123 approval before merge.

File: src/main/java/io/debezium/connector/spanner/config/BaseSpannerConnectorConfig.java [hunk 1]
```diff
@@ -140,7 +142,16 @@ public abstract class BaseSpannerConnectorConfig extends CommonConnectorConfig {
             .withValidation(FieldValidator::isNotBlank)
             .withDescription("Spanner database id");
 
-    public static final Field CHANGE_STREAM_NAME = Field.create(GCP_SPANNER_CHANGE_STREAM_PROPERTY_NAME)
+    public static final Field DATABASE_ROLE = Field.create(GCP_SPANNER_DATABASE_ROLE_PROPERTY_NAME)
+            .withDisplayName("DatabaseRole")
+            .withType(Type.STRING)
+            .withGroup(Field.createGroupEntry(Field.Group.CONNECTOR, 5))
+            .withWidth(Width.SHORT)
+            .withImportance(Importance.HIGH)
```
[the remark this answers is not in the document]
Comment (imtj1) [reply src/main/java/io/debezium/connector/spanner/config/BaseSpannerConnectorConfig.java:150]: Yes, it does. No optional value needed. My integ test did not pass this parameter with the config.

**Comment by jpechane** [comment]
@imtj1 Applied, thanks

**Comment by jpechane** [comment]
@imtj1 Could you please create a docs update PR in the main repo? Thanks a lot.

**Pull request merged**